src/pyramid_debugtoolbar/tbtools.py

```diff
@@ -32,7 +32,7 @@
 from pyramid_debugtoolbar.utils import EXC_ROUTE_NAME
 
 _coding_re = re.compile(br'coding[:=]\s*([-\w.]+)')
-_line_re = re.compile(br'^(.*?)$(?m)')
+_line_re = re.compile(br'(?m)^(.*?)$')
```

Comment:
Please use [verbose regular expressions with a comment](https://docs.python.org/3/library/re.html#re.VERBOSE) that describes what this regex does. We have done a poor job of doing that, and should do better to avoid bugs like this one. Another example: https://stackoverflow.com/a/13852096/2214933

Additionally the other two regexes should be adapted. This tool may be helpful: https://regex101.com/. Thank you!